### DISCUSSION TITLE
can't specify size as (width,height) in add_page()

### DISCUSSION THREAD

**Discussion by Gina-IT** [py-pdf/fpdf2#612] [Q&A]
I need to create pages with different sizes, 
so I tried **add_page(format=(width, height))**,
but I got **TypeError**: add_page() got an unexpected keyword argument 'format'

or if I try **add_page((width,height))**, 
I got **AttributeError**: 'float' object has no attribute 'upper'

I guess add_page() only accept orientation parameter.
I followed this [solution](https://stackoverflow.com/questions/58744527/pyfpdf-cant-add-page-while-specifying-the-size), installing from github master branch, but still not working.

I tried to install fpdf2 by 2 ways
1) "pip install git+https://github.com/PyFPDF/fpdf2.git@master"
2) git clone and  'python setup.py install'

How can I solve it?

**Comment by Lucas-C** [comment]
This is very surprising, as we have a non-regression unit test covering this usage:
* https://github.com/PyFPDF/fpdf2/blob/master/test/test_add_page.py#L11
* resulting PDF: https://github.com/PyFPDF/fpdf2/blob/master/test/add_page_format.pdf

What version of `fpdf2` are you using? (run `pip freeze` to check the version installed)

Could you provide some **minimal code** reproducing your issue please?
If you don't know how to build a minimal reproducible example, please check this tutorial: https://stackoverflow.com/help/minimal-reproducible-example

**Reply by Gina-IT** [reply]
Sorry,
I tried to uninstall fpdf yesterday, but I think it didn't work well.
I found fpdf  remains installed, so I uninstalled again and it worked.
Thank you for your reply.
